portfolio/src/main/webapp/script.js

```diff
@@ -14,8 +14,15 @@
 
 async function getFormMessage() {
   const response = await fetch('/data');
-  const quote = await response.text();
-  document.getElementById('quote-container').innerHTML = quote;
+  const jsonComments = await response.text();
+  var comments = JSON.parse(jsonComments);
+  
+  Object.keys(comments).forEach(function(commentIndex){
+    var node = document.createElement("p");
```

Comment:
Never use `var`. There is no use for it.

Use `const` whenever possible, and `let` if you _**need**_ to change the assignment of the variable. This should be in the style guide.

Reply:
got it, will use consts and let if I need to change the assignment of the variable 😊